Source/Core/Core/ReliableUDPManager/ReliableUnitTest.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "ReliableUDPConnection.h"
+
+class ReliableUnitTest
+{
+public:
+	ReliableUnitTest(sf::IpAddress adr, bool server);
+	~ReliableUnitTest();
+
+	void TestsServer();
+	void TestClient();
+
+	bool TestConnectionS();
+	bool TestConnectionC();
+
+	bool TestSendingRecievingS();
+	bool TestSendingRecievingC();
+
+	bool TestDroppingS();
+	bool TestDroppingC();
+
+	void UnreliableSentTest1();
+	void RecieveUnreliableSentTest1();
+
+	void ReliableSentTest();
+	void ReliableRecieveTest();
+
+private:
+	#define MaxWords  5
```

Comment:
Constants should be upper-cased: `MAX_WORDS`. Also if possible, just make this a static const int.